### ISSUE TITLE
PWD builtin write always in STDOUT

### ISSUE THREAD

**Issue by Vpekdas** [FirePh0enix/minishell#47]
In the following command 
```C
pwd | pwd
``` 
 the output is written twice.

Fix is to return 0 in if (parent_out != -1) 
```C
int	builtin_pwd(t_minishell *msh, int parent_in, int parent_out, t_node *node)
{
	char	*s;
	int		file;

	(void) msh;
	(void) parent_in;
	s = getcwd(NULL, 0);
	if (!s)
		return (ft_fprintf(2, CWDERA CWDERB, "pwd", strerror(errno)), 1);
	file = 1;
	if (parent_out != -1)
	{
		ft_putendl_fd(s, parent_out);
		return (0);
	}
	else if (node->cmd.outfile)
	{
		file = open_outfile(node);
		if (file == -1)
			return (free(s), 1);
	}
	ft_putendl_fd(s, file);
	free(s);
	if (node->cmd.outfile)
		close(file);
	return (0);
}
```

**Comment by FirePh0enix** [comment]
More something like:

```c
int	builtin_pwd(t_minishell *msh, int parent_in, int parent_out, t_node *node)
{
	char	*s;
	int		file;

	(void) msh;
	(void) parent_in;
	s = getcwd(NULL, 0);
	if (!s)
		return (ft_fprintf(2, CWDERA CWDERB, "pwd", strerror(errno)), 1);
	file = 1;
	if (parent_out != -1)
		ft_putendl_fd(s, parent_out);
	else if (node->cmd.outfile)
	{
		file = open_outfile(node);
		if (file == -1)
			return (free(s), 1);
	}
	else
	{
		ft_putendl_fd(s, file);
		if (node->cmd.outfile)
			close(file);
	}
	free(s);
	return (0);
}
```
Or else you will leak `s`

**Comment by FirePh0enix** [comment]
Fixed in 8b45867
